src/main/java/it/rate/webapp/services/GoogleImageService.java

```diff
@@ -52,6 +53,18 @@ public String savePlaceImage(MultipartFile image, Long placeId) throws IOExcepti
     }
   }
 
+  public byte[] getImageById(String imageId) throws IOException {
+
+    try {
+      try (InputStream imageStream =
+          driveService.files().get(imageId).executeMediaAsInputStream()) {
+        return imageStream.readAllBytes();
+      }
+    } catch (IOException e) {
+      throw new IOException("Could not retrieve image from server");

```

Comment:
Maybe not necessary to catch IOException to throw it anyway.

Reply:
whole method changed to implement new exceptions